.bcr/mobile/presubmit.yml

```diff
@@ -0,0 +1,30 @@
+matrix:
+  unix_platform:
+  - debian11
+  - ubuntu2404
+  - macos_arm64
+  bazel:
+  - 7.x
+  - 8.x
+  - 9.*
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ unix_platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - "@envoy_mobile//library/..."
+    build_flags:
+    - "--java_runtime_version=remotejdk_21"
+    - "--cxxopt=-std=c++17"
+    - "--host_cxxopt=-std=c++17"
+  verify_targets_windows:
```

Comment:
i dont think this is neessary esp for mobile - we long since stopped supporting/testing windows